src/js/components/settings/settings.js

```diff
@@ -106,7 +106,8 @@ export const Settings = ({ allowUserManagement, currentUser, hasMultitenancy, hi
 
 const mapStateToProps = state => {
   const currentUser = state.users.byId[state.users.currentUser];
-  const { plan = 'os', trial = false } = state.organization.organization;
+  const plan = 'os';
+  const trial = true;
```

Comment:
this is probably left over I think...

Reply:
Yes, this and the previous comment are leftovers. Careless committing from me, sorry!